src/instana/instrumentation/aio_pika.py

```diff
@@ -1,7 +1,6 @@
 # (c) Copyright IBM Corp. 2025
 
 try:
-    import aio_pika
```

Comment:
Please do not remove the library imports; otherwise, we will not detect the packages we support during runtime. Instead, use the `# noqa: F401` pragma.

```python
    import aio_pika  # noqa: F401
```